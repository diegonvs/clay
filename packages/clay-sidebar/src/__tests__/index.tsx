/**
 * Â© 2019 Liferay, Inc. <https://liferay.com>
 *
 * SPDX-License-Identifier: BSD-3-Clause
 */

import {cleanup, render} from '@testing-library/react';
import React from 'react';

import ClaySidebar from '../../src';

describe('ClaySidebar', () => {
	afterEach(cleanup);

	it('renders', () => {
		const {container} = render(
			<ClaySidebar />
		);

		expect(container).toMatchSnapshot();
	});
});
