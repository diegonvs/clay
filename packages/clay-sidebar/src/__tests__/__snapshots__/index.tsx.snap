// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClaySidebar renders 1`] = `
<div>
  <div
    class="sidenav-fixed sidenav-fixed-end"
  >
    <ul
      class="tbar-nav"
    />
  </div>
</div>
`;
